en/sensor/avanon_laser_interface.md

```diff
@@ -0,0 +1,56 @@
+# Avionics Anonymous Laser Altimeter Interface
+
+The [Avionics Anonymous Laser Altimeter Interface](https://www.tindie.com/products/avionicsanonymous/uavcan-laser-altimeter-interface/) allows a [number of common rangefinders](#supported_rangefinders) to be connected via the UAVCAN bus (this is a more robust interface than I2C).
+
+## Where to Buy
+
+* [AvAnon Laser Interface](https://www.tindie.com/products/avionicsanonymous/uavcan-laser-altimeter-interface/)
+
+## Supported Rangefinders {#supported_rangefinders}
+
+A full list supported rangefinders can be found on the link above. 
+
+At time of writing the following rangefinders are supported:
+
```

Comment:
FYI, I added this list to make it more obvious what is supported. You might want to add a link to this page in http://docs.px4.io/master/en/sensor/sfxx_lidar.html